index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>A Tribute Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main id="main">
+        <h1 id="title">
+            Dr. Norman Borlaug
+        </h1>
+        <figure id="img-div">
+            <img style = "max-width:100%; display:block;"id="image" src=" https://www.worldfoodprize.org/documents/filelibrary/images/dr_borlaug/The_Rockefeller_Foundation_created__094A02E9447A2.jpg"
+            alt="It's an image of Dr.Noman Borlaug with his team of scientists/researchers.">
+            <figcaption id="img-caption">
+            "Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger."
+            </figcaption>
+        </figure>
+        <p id="tribute-info">
```

Comment:
Please use class instead of id. Using class will enable you to use that selector more than once.

Reply:
This is just a repetition of the first one. Kindly try to add another inline comment to help you practice and understand the project more. :pray: 
You can consider using emojis/icons to give your review a more human-friendly touch. :face_with_spiral_eyes: